tests/tck/features/delete/DeleteTag.IntVid.feature

```diff
@@ -41,6 +41,7 @@ Feature: Delete int vid of tag
       """
     Then the result should be, in any order:
       | player.name | player.age |
+      | EMPTY       | EMPTY      |
```

Comment:
We shouldn't return a result with all properties empty value when the vertex doesn't exist. The empty response is what we expect.

Reply:
Because we add a `vertexKey`, `Delete Tag` will not delete vertex at same time. Vertex is still existed

Reply:
Do we plan to support `fetch tags on hash("Tim Duncan")` later?
This seems like a natural user demand scenario bcz TagKey and VertexKey are abstracted out.